Cargo.toml

```diff
@@ -32,8 +32,9 @@ defmt = { version = "0.3", optional = true }
 futures = { version = "0.3", default-features = false, features = ["async-await"], optional = true }
 
 [dependencies.smoltcp]
-version = "0.10"
+version = "0.11"
 default-features = false
+features = [ "proto-ipv4", "defmt" ]
```

Comment:
Please remove these features. Compilation should work if you also update the version of `smoltcp` in `dev-dependencies`.